src/logic/safe/transactions/gas.ts

```diff
@@ -217,32 +229,35 @@ export const estimateGasForTransactionCreation = async (
     const safeInstance = await getGnosisSafeInstanceAt(safeAddress)
 
     const estimateData = safeInstance.methods.requiredTxGas(to, valueInWei, data, operation).encodeABI()
+    const threshold = await safeInstance.methods.getThreshold().call()
+
+    const fixedGasCosts = getFixedGasCosts(Number(threshold))
+
     const gasEstimationResponse = await getGasEstimationTxResponse({
       to: safeAddress,
       from: safeAddress,
       data: estimateData,
-      gas: safeTxGas ? safeTxGas : undefined,
+      gas: safeTxGas ? safeTxGas + fixedGasCosts : undefined,
```

Comment:
@dasanra  code looks good. 
This is the only line that makes me doubt, we are doing the estimation with something different from what the user entered by hand, safeTxGas is part of the tx, so if we are adding more gas to estimate it and is successful because of the addition of fixedGasCost when the tx is really executed it will fail. 

If the user enters something by hand, he should be sure that it includes fixedGasCost.

Reply:
The gas we have to provide there is undefined or "gasLimit" value